packages/build/src/core/build.js

```diff
@@ -0,0 +1,554 @@
+import { getErrorInfo } from '../error/info.js'
```

Comment:
Most of this file was simply extracted out of `packages/build/src/core/main.js`, so that it could be reused by different entry points.

I'll add comments to the lines that have actually changed.